source/core/coverage/ut_coverage.pkb

```diff
@@ -30,11 +30,12 @@ create or replace package body ut_coverage is
   end;
 
   function get_cov_sources_sql(a_coverage_options ut_coverage_options, a_skip_objects ut_object_names) return varchar2 is
-    l_result                varchar2(32767);
+    l_result                varchar2(32767); --QUESTION: Given fact that we can pass large regex filters same as objects filter should we consider a clob ?
```

Comment:
Let's keep the variables as they are.
Let's pass the filters as parameters instead of concatenating them into the SQL query